kedro/framework/cli/starters.py

```diff
@@ -553,3 +587,19 @@ def _validate_config_file(config: dict[str, str], prompts: dict[str, Any]):
             f"'{config['output_dir']}' is not a valid output directory. "
             "It must be a relative or absolute path to an existing directory."
         )
+
+
+def _validate_config_file_inputs(config: dict[str, str]):
```

Comment:
I think it would be helpful for this method to have a docstring explaining what it does. As we discussed on a call, I was initially confused about the purpose of the method.